docs/installation/ubuntulinux.md

```diff
@@ -31,26 +34,73 @@ Kernels older than 3.10 lack some of the features required to run Docker
 containers. These older versions are known to have bugs which cause data loss
 and frequently panic under certain conditions.
 
-To check your current kernel version, open a terminal and use `uname -r` to display
-your kernel version:
+To check your current kernel version, open a terminal and use `uname -r` to
+display your kernel version:
 
     $ uname -r
     3.11.0-15-generic
 
->**Caution** Some Ubuntu OS versions **require a version higher than 3.10** to
->run Docker, see the prerequisites on this page that apply to your Ubuntu
->version.
+>**Note**: If you previously installed Docker using `apt`, make sure you update
+your `apt` sources to the new Docker repository.
 
+### Update your apt sources
 
-### For Vivid 15.04
+Docker's `apt` repository contains Docker 1.7.1 and higher. To set `apt` to use
+packages from the new repository:
 
-There are no prerequisites for this version.
+1. If you haven't already done so, log into your Ubuntu instance.
 
-### For Trusty 14.04
+2. Open a terminal window.
 
-There are no prerequisites for this version.
+3. Add the new `gpg` key.
 
-### For Precise 12.04 (LTS)
+        $ apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+
+4. Open the `/etc/apt/sources.list.d/docker.list` file in your favorite editor.
+
+    If the file doesn't exist, create it.
+
+5. Remove any existing entries.
+
+6. Add an entry for your Ubuntu operating system.
+
+    The possible entries are:
+
+        # Ubuntu Precise
+        deb https://apt.dockerproject.org/repo ubuntu-precise main
+        # Ubuntu Trusty
+        deb https://apt.dockerproject.org/repo ubuntu-trusty main
+        # Ubuntu Utopic
+        deb https://apt.dockerproject.org/repo ubuntu-utopic main
+        # Ubuntu Vivid
+        deb https://apt.dockerproject.org/repo ubuntu-vivid main
+        # Ubuntu Wily
+        deb https://apt.dockerproject.org/repo ubuntu-wily main
+
+7. Save and close the `/etc/apt/sources.list.d/docker.list` file.
+
+8. Update the `apt` program.
+
+        $ apt-get update
+
+9. Purge the old repo if it exists.
+
+        $ apt-get purge lxc-docker*
+
+10. Verify that `apt` is pulling from the right repository.
+
+        $ apt-get policy docker
```

Comment:
`apt-cache policy docker-engine` (I think?)

Same comment here as for the Debian instructions